### DISCUSSION TITLE
Polar plot with Errorbars

### DISCUSSION THREAD

**Discussion by DL9HDA** [jfree/jfreechart#281] [Q&A]
Hello, I am currently working with version 1.5.3 and I need to create a polar plot. I currently plot my measured values in red and I have two other circles in black without no filling (.setSeriesFilled(1, false)), which represent the tolerance ranges. Between these two circles are normally my measured values. The currently used renderer is the DefaultPolarItemRenderer(). 

Question: Is it possible to display my measured values (red) with error bars (with the orientation from the plot centre to the outside)? Surely this looks a bit strange, but it would be quite helpful for my presentation. Here is an example: https://i.stack.imgur.com/ZUMy8.png

Best regards
Holger

**Comment by trashgod** [comment]
At a guess, you'd need a custom renderer. You might look among the examples examined [here](https://stackoverflow.com/search?tab=votes&q=%5bjfreechart%5d%20polar).
